libs/api/src/lib/graphql/article.ts

```diff
@@ -37,7 +37,7 @@ import {GraphQLTag} from './tag/tag'
 
 export const GraphQLArticleFilter = new GraphQLInputObjectType({
   name: 'ArticleFilter',
-  fields: {
+  fields: () => ({
```

Comment:
Resolves circular dependency with blocks. Got to be a problem because of memberPlan now importing pages.